discoveryengine/answer_query_sample.py

```diff
@@ -20,6 +20,8 @@
 from google.cloud import discoveryengine_v1 as discoveryengine
 
 # TODO(developer): Uncomment these variables before running the sample.
+# NOTE: This sample shows all available answer phase options. Remove any options that you don't plan to use.
```

Comment:
nit: would recommend something a little more clear and specific for this comment. Suggestion: 

# The `answer_generation_spec` below includes all available answer phase options. 
# Modify or remove any options that are not relevant to your use case to optimize 
# the answer generation process.